chainntnfs/txnotifier.go

```diff
@@ -887,9 +887,12 @@ func (n *TxNotifier) dispatchSpendDetails(ntfn *SpendNtfn, details *SpendDetail)
 //   confirmation registration for.
 //
 // In the event that the transaction is relevant, a confirmation/spend
-// notification will be dispatched to the relevant clients. Confirmation
-// notifications will only be dispatched for transactions that have met the
-// required number of confirmations required by the client.
+// notification will be queued for dispatch to the relevant clients.
+// Confirmation notifications will only be dispatched for transactions that have
+// met the required number of confirmations required by the client.
+//
+// NOTE: In order to actually dispatch the relevant transaction notifications to
+// clients, NotifyHeight must be called.
```

Comment:
should probably note that this method MUST be called if `ConnectTip` succeeds in order to maintain correctness of the notifications. the current godoc makes it sounds semi-optional imo

Reply:
Fixed.